Sources/Client/Model/Channel/Channel.swift

```diff
@@ -222,16 +222,12 @@ public final class Channel: Codable {
     }
 }
 
-extension Channel: Hashable, CustomStringConvertible {
+extension Channel: Equatable, CustomStringConvertible {
     
     public static func == (lhs: Channel, rhs: Channel) -> Bool {
         lhs.cid == rhs.cid
     }
     
-    public func hash(into hasher: inout Hasher) {
-        hasher.combine(cid)
-    }
-    
```

Comment:
This is custom implementation, why did we remove them?

Reply:
It's no need to be `Hashable`. We don't use it.